### DISCUSSION TITLE
Character editor doesnt detect any characters

### DISCUSSION THREAD

**Discussion by NikRetaNCAM** [CodenameCrew/CodenameEngine#189] [Q&A]
I open the character editor by pressing 7 in the main menu, and I see this
![image](https://github.com/FNF-CNE-Devs/CodenameEngine/assets/60806448/80c0bd9e-96f1-4cfa-a34f-16bd6c223be5)
I dont get it

**Comment by charlesisfeline** [comment]
did you just remove all of the characters

**Reply by NikRetaNCAM** [reply]
nope, them all are there

**Comment by NexIsDumb** [comment]
i mean this should be fixed by now but why did you made it a discussion??
